trillian/ctfe/instance.go

```diff
@@ -55,6 +51,39 @@ func LogConfigFromFile(filename string) ([]*configpb.LogConfig, error) {
 	return cfg.Config, nil
 }
 
+// MultiLogConfigFromFile creates a LogMultiConfig proto from the given
+// filename, which should contain text-protobuf encoded configuration data.
+// Does not do full validation of the config but checks that it is non empty.
+func MultiLogConfigFromFile(filename string) (*configpb.LogMultiConfig, error) {
+	cfgText, err := readConfig(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	var cfg configpb.LogMultiConfig
+	if err := proto.UnmarshalText(string(cfgText), &cfg); err != nil {
+		return nil, fmt.Errorf("failed to parse multi-backend log config: %v", err)
+	}
+
+	if len(cfg.LogConfigs.Config) == 0 || len(cfg.Backends.Backend) == 0 {
+		return nil, errors.New("config is missing backends and/or log configs")
+	}
+	return &cfg, nil
+}
+
+func readConfig(f string) ([]byte, error) {
```

Comment:
Not sure `readConfig` adds anything -- could just use `ioutil.ReadFile` directly as it already gives different errors for empty argument ("open : Invalid argument") and file-not-found ("open filename: No such file or directory").

Reply:
Done.